qubes/storage/lvm.py

```diff
@@ -347,6 +387,22 @@ def _is_snapshot(self):
     def _is_volatile(self):
         return not self.snap_on_start and not self.save_on_stop
 
+    @property
+    def size(self):
```

Comment:
It cause ThinVolume fail to load, because Volume.__init__ tries to set `self.size = int(size)`, but there is no setter implemented.